Source/missiles.cpp

```diff
@@ -383,7 +383,7 @@ bool Plr2PlrMHit(int pnum, int p, int mindam, int maxdam, int dist, missile_id m
 		dam = target._pHitPoints / 3;
 	} else {
 		dam = mindam + GenerateRnd(maxdam - mindam + 1);
-		if (MissilesData[mtype].mType == 0)
+		if (MissilesData[mtype].mType == 0 && resper == 0)

```

Comment:
Do we want to apply bonus damage when the player has no resistance to the element? Maybe we can use `mir` again?
```suggestion
		if (MissilesData[mtype].mType == 0 && MissilesData[mtype].mResist == MISR_NONE)
```